formserver/.gitignore

```diff
@@ -0,0 +1,26 @@
+.gradle
```

Comment:
What happens if we have `.gitignore` not at the repo root? What does it do?

Reply:
Hmm. Spring boot generation probably put this here. I wanna say it does nothing. I'll see if there is anything not already in the main

Reply:
ill fix this in this pr

Reply:
done